### PR TITLE
[TITO-17]

### PR DESCRIPTION
**Pull request by whitneyjsmith**

### CODE REVIEW

**Comment by dahjah** [comment]
Looks awesome so far!

As we were discussing during the meeting, we decided that I think we will actually want to remove the attribute_bases manytomany field on the Product. :)

In addition to that change, can you also add __str__ methods to the models, and then use filter_horizontal in the admin registration for Product?
(That would look something like this:

```
@admin.register(Product)
class ProductAdmin(admin.ModelAdmin):
    filter_vertical = ('attributes',)
```
)

**Review by kpavia — COMMENTED** [review]
I second DJ's comment!
Looks good but we just had these things occur to us while checking it out and discussing.

File: products/admin.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 from django.contrib import admin
 
-from .models import Product
+from .models import Product, Attribute, AttributeBase
 
 admin.site.register(Product)
```
Comment (dahjah) [inline products/admin.py]: Would you mind updating this to look something like this? 

```
@admin.register(Product)
class ProductAdmin(admin.ModelAdmin):
    filter_vertical = ('attributes',)
```

File: products/models.py [hunk 2]
```diff
@@ -2,8 +2,20 @@
 from django.core.files.storage import default_storage
 
 
+class AttributeBase(models.Model):
+    name = models.CharField(max_length=75)
+
```
Comment (dahjah) [inline products/models.py:7]: Can you add some __str__ methods here? :)

**Review by dahjah — CHANGES_REQUESTED** [review]

**Comment by whitneyjsmith** [comment]
@dahjah @kpavia awesome I made those changes. Thanks for reviewing my code

**Review by dahjah — APPROVED** [review]
Looks great!

**Review by kpavia — APPROVED** [review]

**Pull request merged**